compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.td

```diff
@@ -499,6 +499,28 @@ def IREEGPU_TargetAttr : AttrDef<IREEGPU_Dialect, "Target"> {
     bool supportsTF32InputMMAOps() const;
     // Returns true if this target supports TensorCore synchronized MMA ops.
     bool supportsSyncMMAOps() const;
+    // Returns the SKU of the target GPU if available.
+    std::optional<StringRef> getSKU() const {
+        llvm::StringRef arch = getArch();
```

Comment:
I think we should move this code somewhere close to or in `KnownTargets.cpp`; The generic GPU attributes shouldn't know anything about the rocm backend details and hip targets.